test/e2e/framework/helper/certificates.go

```diff
@@ -280,17 +267,6 @@ func (h *Helper) deduplicateExtKeyUsages(us []x509.ExtKeyUsage) []x509.ExtKeyUsa
 	return us
 }
 
-func (h *Helper) WaitCertificateIssued(ns, name string, timeout time.Duration) error {
```

Comment:
Why is this function being deleted?

Reply:
The function did almost the same as `WaitForCertificateReady`, so it could be replaced by this function.